### PR TITLE
chore(v3): update with ethers wrap

### PR DESCRIPTION
**Pull request by cbrzn**

### CODE REVIEW

File: v3/wrap/README.md [hunk 1]
```diff
@@ -172,15 +172,15 @@ We used the Ethereum wrapper's `encodeFunction` method to encode calldata for Un
 // An abridged copy of src/router/utils.ts in the Uniswap v3 wrapper
 
 import {
```
Comment (dOrgJelli) [inline v3/wrap/README.md:174]: NIT: we should probably inject these code snippets using doc-snippets, or just point the reader to source files (src/ or examples/)

File: v3/wrap/client-config.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import {
```
Comment (dOrgJelli) [inline v3/wrap/client-config.ts:1]: Random thought: what do you think about naming this file "polywrap.config.ts"?

File: v3/wrap/client-config.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            mainnet: new Connection({
+              provider: "http://localhost:8546",
+              signer: defaultSigner,
+            }),
+          },
+          defaultNetwork: "mainnet",
+        }),
+      })
+    )
+    .addRedirect(
```
Comment (dOrgJelli) [inline v3/wrap/client-config.ts]: What is this redirect needed for? Maybe add a comment?

File: v3/wrap/client-config.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      "wrap://ens/ethers.wraps.eth:utils@0.0.1",
+      "wrap://ipfs/QmRENy16y3p4VTXv7jXyVrPhJ9gdTMK8U8sXmLSTXW5ReF"
+    )
+    .addRedirect(
```
Comment (dOrgJelli) [inline v3/wrap/client-config.ts]: Same as above, why is this needed?

**Review by dOrgJelli — APPROVED** [review]
Looks great! Just concerned about the redirects, but other than that 👍

File: v3/wrap/README.md (hunk 1 above)
Comment (cbrzn) [reply v3/wrap/README.md:174]: i think this should somehow be fixed with the new wraps-doc approach...

File: v3/wrap/client-config.ts (hunk 2 above)
Comment (cbrzn) [reply v3/wrap/client-config.ts:1]: i feel like that name might be confusing with any toolchain manifest file :thinking: but I am not close to that tho

File: v3/wrap/client-config.ts (hunk 3 above)
Comment (cbrzn) [reply v3/wrap/client-config.ts]: they were not added to ens when i did the commit, going to change it
